ReactAndroid/src/main/java/com/facebook/react/uimanager/Spacing.java

```diff
@@ -192,6 +209,9 @@ private static float[] newFullSpacingArray() {
       YogaConstants.UNDEFINED,
       YogaConstants.UNDEFINED,
       YogaConstants.UNDEFINED,
+      YogaConstants.UNDEFINED,
```

Comment:
Is it possible to specify in the type that the array is of size `Spacing.ALL` so that when we add an enum, we are forced to update this?